File: modul-326/bomberman-modul-326/src/main/java/ch/gibb/bomberman/protocol/server2client/PlayerMoved.java
package ch.gibb.bomberman.protocol.server2client;

import ch.gibb.bomberman.network.Message;
import ch.gibb.bomberman.protocol.Direction;

public class PlayerMoved implements Message {
  private String playerName;
  private Direction direction;

  public PlayerMoved(String playerName, Direction direction) {
    this.playerName = playerName;
    this.direction = direction;
  }

  public String getPlayerName() {
    return playerName;
  }

  public Direction getDirection() {
    return direction;
  }
}

File: modul-326/bomberman-modul-326/unity/Bomberman_Returns/Assets/Models/ReceiveMessages/PlayerMoved.cs
using System.Collections;
using System.Collections.Generic;

public class PlayerMoved : Message
{
    private string playerName;

    private Direction direction;

    public PlayerMoved(string playerName, Direction direction)
    {
        this.playerName = playerName;
        this.direction = direction;
    }

    public Direction Direction { get => direction; }
    public string PlayerName { get => playerName; }

}
